kokoro/scripts/macos/build.sh

```diff
@@ -24,6 +24,9 @@ BUILD_ROOT=$PWD
 SRC=$PWD/github/SPIRV-Tools
 BUILD_TYPE=$1
 
+# Fix dubious ownership issues on Kokoro.
```

Comment:
A better comment like you have for the wasm build would be nice. This comment is not helpful to me.

Reply:
propagated wasm comment to all.